### PR TITLE
Add image for PHP 8.0 and delete support for PHP 7.1

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add image for PHP 8.0 and delete support for PHP 7.1
| Type?             | improvement
| BC breaks?        | yes / no
| Deprecations?     | yes / no
| Fixed ticket?     | 
| How to test?      | Run the new image
| Possible impacts? |

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-versions: ['7.1', '7.2', '7.3', '7.4']
+        php-versions: ['7.2', '7.3', '7.4', '8.0']
```
Comment (atomiix) [inline .github/workflows/docker.yml]: You can add PHP 8.1 now :)

Comment (matks) [inline .github/workflows/docker.yml]:
```suggestion
        php-versions: ['7.2', '7.3', '7.4', '8.0', '8.1']
```
ping @boubkerbribri 😉

**Comment by boubkerbribri** [comment]
@matks , @atomiix it fails because we don't the base image here https://github.com/PrestaShop/docker/tree/master/base/images

**Review by atomiix — APPROVED** [review]

**Comment by atomiix** [comment]
Thanks @boubkerbribri!

**Pull request merged**